src/3d/3d_primitives.js

```diff
@@ -520,4 +520,4 @@ p5.prototype.box = function(width, height, depth){
 
 };
 
-module.exports = p5;
\ No newline at end of file
+module.exports = p5;
```

Comment:
weird, i was editing this through the github web UI and didn't add this newline...